be/src/exprs/agg/array_agg.h

```diff
@@ -32,20 +32,20 @@ namespace starrocks {
 struct ArrayAggAggregateState {
     void update(FunctionContext* ctx, const Column& column, size_t index, size_t offset, size_t count) {
         if (UNLIKELY(index >= data_columns->size())) {
-            ctx->set_error(
-                    (std::string(fmt::format("index {} is larger than column size {}.", index, data_columns->size())))
-                            .c_str(),
-                    false);
+            if (ctx != nullptr) {
```

Comment:
why change this in this PR

Reply:
Because of compiler warning, I already commented.